src/pages/javascript/conditional-ternary-operators/index.md

```diff
@@ -1,15 +1,26 @@
 ---
 title: Conditional Ternary Operators
 ---
-## Conditional Ternary Operators
-
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/javascript/conditional-ternary-operators/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
-
-<a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
-
-<!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
-
-#### More Information:
-<!-- Please add any articles you think might be helpful to read before writing the article -->
-
-
+## Basic usage
```

Comment:
Can we have this as the article title please, you can place Basic usage on the next line with `###`